pkg/cmd/plugincheck2/main.go

```diff
@@ -38,12 +38,6 @@ func main() {
 	logme.Debugln("source code: ", *sourceCodeUri)
 	logme.Debugln("archive file: ", flag.Arg(0))
 
-	if *configFlag == "" {
```

Comment:
Instead of failing for not passing a default config we assume  a basic default config.